packages/chrome.extensions.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,64 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+
+  $extensions = @(
+    # MetaMask
+    'nkbihfbeogaeaoehlefnkodbefgpgknn'
+    # Phantom
+    'bfnaelmomeimhlpmgjnjophhpkkoljpa'
+    # BNB Chain Wallet
+    'fhbohimaelbohpjbbldcngcnapndodjp'
+    # Avira Password Manager
+    'caljgklbbfbcjjanaijlacgncafpegll'
+    # KeePassXC-Browser
+    'oboonakemofpalcgghocfoadofidjkkk'
+    # Yoroi
+    'ffnbelfdoeiohenkjibnmadjiehjhajb'
+    # XDEFI Wallet
+    'hmeobnfnfcmdkdcmlblgagmfpfboieaf'
+    # ...
+  )
+
+  $regKeyPath = "HKLM:\SOFTWARE\WOW6432Node\Policies\Google\Chrome\ExtensionInstallForcelist"
+  $updateUrl = "https://clients2.google.com/service/update2/crx"
+
+  New-Item -Path $regKeyPath -Force -ea 0 | Out-Null
+  $valueName = 1
+  Foreach ($extensionId in $extensions)
+  {
+    New-ItemProperty -Path "$regKeyPath" -Name "$valueName" -Type String -Value "$extensionId;$updateUrl" -Force -ea 0 | Out-Null
+    $valueName += 1
+  }
+
+  $chromePath = "${Env:ProgramFiles}\Google\Chrome\Application\chrome.exe"
+  $extensionsDir = "${Env:LocalAppData}\Google\Chrome\User Data\Default\Extensions"
+
+  $loaded = $false
+  while (-not $loaded)
+  {
+    # Restart Chrome.
+    Stop-Process -Force -Name Chrome -ea 0
+    $chromeId = (Start-Process -FilePath $chromePath -passthru).id
+
+    # Wait for the extensions to be loaded.
+    Start-Sleep -Seconds 120
+    Stop-Process -Force -Id $chromeId | Out-Null
+
+    # Make sure all of the extensions are loaded.
+    $loaded = $true
+    Foreach ($extensionId in $extensions)
+    {
+      $extensionPath = Join-Path $extensionsDir $extensionId
+      if (-not (Test-Path -Path $extensionPath))
+      {
+        $loaded = $false
+        break
+      }
+    }
+  }
```

Comment:
I have two more concerns:
- can we lower the 120 seconds? as we are checking if the extensions are installed and this time increases installation time if not needed.
- What happens if there is an error and the extensions never install? I think we should have a reasonable maximum tries in the loop and fail the package to avoid it never finishes.